help/visualize/viewers/chord.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Chord"
+---
+A chord diagram is a graphical method of displaying the inter-relationships
+between data.
+You can use it to visualize complex networks or flows between
+different entities.
+Chrod diagram arranges data radially around a circle and draws the flows
+between the data points as arcs connecting the data.
+The chord width represents the strength of the flow
+between the entities.
+
+![Chord viewer](chord-viewer.png)
+
+## Create a Chord viewer
+
+To create a **Chord** viewer, open the main menu and select 
+the option "Add -> JavaScript Viewers -> Chord"
+
+When you add a chord viewer in Datagrok, the platform selects three columns:
+
+* A categorical "From" column
+* A categorical "To" column
+* A numeric "Chord length" column representing the flow strength between source
+and target categories
+
+## Configuring a Chord viewer
+
+You can set the source, target and "Chord length" columns
+and customize a visualization options.
+To do that, click the **Gear** icon on top of the viewer and use the **Data**
+and **Misc** info panels 
+on the **Context Pane** to manage the viewer’s settings.
+
+For example, you can:
+
+* Change the aggregation type of connections via **Agg type**: just count of
+connections or sum of the "Chord length" values.
+* Set the connection color mode by source or by target via **Color by** control.
```

Comment:
the style of the bullet points should be consistent. Pick one style (e.g., [lead-in sentence] : [options], which is what you use in bullet point no.1, or descriptive (as used in bullet point no.2) and apply it to all.

Reply:
Fixed